Assets/Countly/Example/Scripts/UserDetail.cs

```diff
@@ -30,12 +30,11 @@ public async void SetUserDetail()
 
     public async void SetCustomUserDetail()
     {
-        Dictionary<string, object> userCustomDetail = null;
```

Comment:
this is probably a small 'bad value' test to see if things break. Can you check when this was added?

Reply:
It's added back when we change the structure of the example app. Tbh I don't remember the reason it was there but since K merged it I believe it was a small bad value to see whether things break or not. Still, I don't think it's adding all that much of a value within the example app. That's why I believe we can remove it.

Reply:
sure seems like it was copy pasted from UserCustomDetailsTests you can erase